desktop_version/src/FileSystemUtils.cpp

```diff
@@ -78,15 +78,13 @@ void FILESYSTEM_init(char *argvZero)
 #endif
 	if (!PHYSFS_mount(output, NULL, 1))
 	{
-		SDL_ShowSimpleMessageBox(
-			SDL_MESSAGEBOX_ERROR,
-			"data.zip missing!",
-			"You do not have data.zip!"
-			"\n\nGrab it from your purchased copy of the game,"
-			"\nor get it from the free Make and Play Edition.",
-			NULL
-		);
+		puts("Error: data.zip missing!");
```

Comment:
Let's do the `puts` lines first, then the messagebox afterward (SDL_messagebox is written to be safe anywhere in the program, but for Linux it may require some X ugliness to successfully load).

Other than that, looks good to me! If you want to be the one to write `desktop_version/CONTRIBUTORS.txt` with your name in it, that's fine as well.

Reply:
Sounds like a plan